app/views/items/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Shop</title>
+	</head>
+	<body>
+		<h3>Items:</h3>
+		<div>
+			<% @items.each do |item| %>
+			<div>
+				<p><strong><%= link_to item.title, item_path(item) %></strong></p>
+				<p>Description: <%= item.description %></p>
+				<p>Price:$ <%= item.price %></p>
+				<img src=<%=item.image%>>
+			</div>
+				<% end %>
+		</div>
+	</body>
+</html>
```

Comment:
Let's remove the `<html>` and `<head>` element because we'll have this persisting in all our views via the `application.html.erb` file.

Reply:
The <html> one I can, but I need the <head> tag to name the tab based on data given by each individual controller